.github/workflows/test.yml

```diff
@@ -57,25 +57,42 @@ jobs:
   linkcheck:
     name: Build site and check links
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
         with:
           submodules: recursive
-      - run: make build
       - uses: dart-lang/setup-dart@fedb1266e91cf51be2fdb382869461a434b920a3
         with:
           sdk: stable
-      - name: Fetch packages
+      - name: Fetch Dart packages
         run: dart pub get
-      - name: Check for broken Markdown links
-        run: dart run dart_site check-link-references
-      - name: Validate the firebase.json file
-        run: dart run dart_site verify-firebase-json
+      - uses: pnpm/action-setup@ebcfd6995dade4b0104ac774445cef8b3b4635b0
+        with:
+          version: 8
       - uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8
         with:
           node-version: ${{ env.NODE_VERSION }}
-      - run: npm install -g firebase-tools@13.0.2
+          cache: 'pnpm'
+      - name: Install node dependencies
+        run: pnpm install
+      - name: Build site
+        run: dart run dart_site build
+      - name: Check for broken Markdown links
+        run: dart run dart_site check-link-references
       - name: Check internal site links are functional
         run: dart run dart_site check-links
```

Comment:
dart_site references the tooling that's in tool/dart_site right?

Reply:
Yep! This will be consolidated under a simpler one word command in the future, like `dart_site check-links` or something.

Reply:
PERFECT!